.github/workflows/test.yml

```diff
@@ -1,8 +1,9 @@
 name: Run tests
 on:
   workflow_dispatch:
+  push:
   pull_request:
-    branches:  [ main ]
+    branches:  [ main, ethers-v5 ]
```

Comment:
CI for `ethers-v5` branch is defined [there](https://github.com/zksync-sdk/zksync-ethers/blob/ethers-v5/.github/workflows/test.yml#L5). CI is run on every commit in PR that's why we are `synchronize` in our types:
`types: [ opened, reopened, synchronize ]`